app/(header-default)/labels-certificats/[slug]/page.tsx

```diff
@@ -97,8 +83,7 @@ const LabelsAndCertificatsPage = async (props: AppRouterProps) => {
           session={session}
         />
         {!checkHasLabelsAndCertificates(uniteLegale) &&
-          (!protectedCertificats ||
-            !hasProtectedCertificatsEntreprise(protectedCertificats)) && (
+          !hasRights(session, EScope.protectedCertificats) && (
```

Comment:
I feel like these conditions dont work as we will miss the `<p>Cette structure ne possède aucun label ou certificat.</p>` in case we have `EScope.protectedCertificats` but no answer for protectedCertificats

Then
- either we do a full server render for this page, like dirigeant
- either we use API routes as they seem more relevant to this multiple calls architecture